release/uname_linux_test.go

```diff
@@ -1,73 +0,0 @@
-// -*- Mode: Go; indent-tabs-mode: t -*-
-
-/*
- * Copyright (C) 2017 Canonical Ltd
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 3 as
- * published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- *
- */
-
-package release_test
-
-import (
-	"syscall"
-
-	. "gopkg.in/check.v1"
-
-	"github.com/snapcore/snapd/release"
-)
-
-func (s *ReleaseTestSuite) TestKernelVersion(c *C) {
-	ver := release.KernelVersion()
-	// Ensure that we got something.
-	c.Check(ver, Not(Equals), "")
-}
-
-func (s *ReleaseTestSuite) TestGetKernelRelease(c *C) {
```

Comment:
This seems like an useful test to keep. Should we save it?

Reply:
done